docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -65,13 +65,13 @@ Currently, Codacy sends coverage data to your Git provider from both the old and
 
 ## If you are using the old Coverage status check to block merging pull requests on GitHub
 
-If you are using the old status check to block merging pull requests on GitHub, [edit the branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#editing-a-branch-protection-rule) of your pull requests' target branch under the section **Require status checks to pass before merging**:
+If you are using the old status check to block merging pull requests on GitHub, [remove the old Codacy Coverage report as a required check by editing the branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#editing-a-branch-protection-rule) of your pull requests' target branch under the section **Require status checks to pass before merging**:
```

Comment:
Splitting into two steps, for clarity.

```suggestion
If you are using the old status check to block merging pull requests on GitHub, remove the old Codacy Coverage report as a required check of your pull requests' target branch:

1.  [Edit the branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#editing-a-branch-protection-rule) of your pull requests' target branch.
1.  Remove the old **Codacy Coverage Report** under the section **Require status checks to pass before merging**:
```

Reply:
I'm not sure we should split here, since by "removing the old Codacy Coverage Report" status check, the user is "editing the branch protection rules", so these two steps are really one action.

At most, the first step should be "Navigate to the branch protection rules page" or similar. Thoughts?